### DISCUSSION TITLE
why aren't there more items in the Microsoft catalog

### DISCUSSION THREAD

**Discussion by rbarbrow** [microsoft/winget-cli#1194] [General] [answered]
i recently needed to install visio and today I wanted to try out power automate desktop  neither are in the winget package library

is this just something that Microsoft has to get around to?

**Comment by denelon** [comment, marked as answer]
Either that, or the community may submit packages. In some cases, installers or licensing restrictions may prevent packages from being submitted / accepted.

**Reply by rbarbrow** [reply]
I mean its Microsoft's tool and Microsoft's software you think if there is one instance where they could have the authority it would be this

**Reply by denelon** [reply]
Visio requires [licensing](https://support.microsoft.com/en-us/office/install-visio-or-access-visio-for-the-web-f98f21e3-aa02-4827-9167-ddab5b025710), and it looks like Power Automate Desktop needs certain [dependencies](#163) which aren't supported in the Windows Package manager yet: [prerequisites](https://docs.microsoft.com/en-us/power-automate/desktop-flows/setup#prerequisites).
